packages/typescriptlang-org/src/components/search/ResultRow.tsx

```diff
@@ -0,0 +1,73 @@
+import { withPrefix } from "gatsby"
+import * as React from "react"
+
+import { cx } from "../../lib/cx"
+import { SearchHit } from "./types"
+
+import "./ResultRow.scss"
+
+export type ResultRowprops = {
+  exactMatch?: boolean
+  installer: [string, string]
+  hit: SearchHit
+}
+
+export const ResultRow: React.FC<ResultRowprops> = ({
+  exactMatch,
+  installer,
+  hit: {
+    description,
+    downloadsLast30Days,
+    humanDownloadsLast30Days,
+    modified,
+    name,
+    types,
+  },
+}) => {
+  const [icon, label] =
+    types.ts === "included"
+      ? ["in", "included"]
+      : ["dt", "from DefinitelyTyped"]
+
+  return (
+    <tr className={cx("resultRow", exactMatch && "resultRowExactMatch")}>
+      <td
+        className={cx(
+          "downloads",
+          downloadsLast30Days > 1_000_000
+            ? "downloadsMillion"
+            : downloadsLast30Days > 100_000 && "downloadsHundredThousand"
+        )}
+      >
+        {humanDownloadsLast30Days}
+      </td>
+      <td className="via">
+        <div
+          aria-label={`Types ${label}`}
+          className={cx("typeImage", icon)}
+          role="img"
+        />
+      </td>
+      <td className="name">
+        <strong>{name}</strong>
+        {description}
```

Comment:
Question: do you have a preference on method to convert this markdown description to plaintext? I would default to `marked` from past experience.

Reply:
Aye, no dependencies for this, I'm open to a rudimentary regex dropping `[xbc](yzx)` to `xbc` but even npm doesn't md parse

![Screen Shot 2020-09-04 at 12 19 11 PM](https://user-images.githubusercontent.com/49038/92262121-a5b15480-eea8-11ea-9daf-d4d19b6b5986.png)